internal/cache/file/downloader/job.go

```diff
@@ -262,7 +265,7 @@ func (job *Job) downloadObjectAsync() {
 				if newReader == nil {
 					newReaderLimit = min(start+sequentialReadSize, end)
 					newReader, err = job.bucket.NewReader(
-						job.cancelCtx,
```

Comment:
I didn't get this change, still we are breaking the loop when job.cancelCtx.Done()?

Reply:
Sorry, I thought I mentioned this offline. if not, then please see the description of https://github.com/GoogleCloudPlatform/gcsfuse/pull/1534. Happy to discuss this offline as well.

Reply:
Yeah, better to keep in separate PR.